api/src/main/java/javax/enterprise/inject/Instance.java

```diff
@@ -203,5 +203,17 @@
      *         destroying bean instances
      */
     public void destroy(T instance);
+    
+    /**
+     * <p>
+     * Instructs the container to destroy the instance. The instance passed should be a dependent scoped bean instance
```

Comment:
Hmm, I see. In that case it makes sense.
Although we might want to name this method so that it indicates the attempt to destroy the instance, wdyt? Something like ```tryToRelease```? Just to draw some clear line between ```destroy``` method and this one from user point of view.